playbooks/acs.yml

```diff
@@ -165,6 +165,14 @@
   tags:
     - search
 
+- name: Populate common facts
+  hosts: all
```

Comment:
let's be specific I trhink we should just have `respository:acc:adw`. That will also avoid includuing external hosts.